README.md

```diff
@@ -39,25 +35,22 @@
 <br/>
 <br/>
 
-
 <!-- ABOUT THE PROJECT -->
 # About The Project
 
-thin-edge.io is the first open-source and cloud-agnostic edge framework designed for resource-constrained IoT edge devices.  
+thin-edge.io is the first open-source and cloud-agnostic edge framework designed for resource-constrained IoT edge devices.
```

Comment:
There is a NBSP between `for` and  `resource-constrained`.

Reply:
nbsp is not markdown standard, it is custom addition for markdown interpreters which support HTML.

Reply:
Shouldn't we delete NBSP?